reposerver/src/main/resources/db/migration/V11__add_delegations.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE `delegations` (
```

Comment:
I would love to have a foreign key on `repo_id`

Reply:
The only table I could link repo_id to is `repo_namespaces` but I am not sure that would make sense. I would need an unique key on repo_namespaces, and while that is ok for now, it would make it impossible for a repo to belong to multiple namespaces in the future...